sparse_strips/vello_cpu/src/dispatch/multi_threaded.rs

```diff
@@ -488,6 +498,28 @@ impl Dispatcher for MultiThreadedDispatcher {
             }
         }
     }
+
+    fn generate_wide_cmd(
+        &mut self,
+        strip_buf: &[Strip],
+        fill_rule: Fill,
+        paint: Paint,
+        thread_idx: u8,
+    ) {
+        // Note that we are essentially round-tripping here: The wide container is inside of the
+        // main thread, but we first send a render task to a child thread which basically just
+        // forwards it back to the main thread again. We cannot apply the wide command directly
+        // here because there might be other paths that are currently being processed in a child
+        // thread that should be rendered _before_ this wide command. Therefore, we treat it like
+        // any other render task so that we can utilize the same mechanism of assigning a task ID
+        // to ensure that they are executed in order.
```

Comment:
My main question is how the order is preserved: is it due to using an ordered channel with task id assignments, which ensures the generated wide tiles maintain execution order?

Reply:
Gotcha, thanks for confirming! This is why we use time cost estimations, to minimize situations where threads are waiting on each other, right? It’s really interesting to see how recordings will be properly supported in a multithreaded environment.